dds_cli/__main__.py

```diff
@@ -192,9 +192,10 @@ def dds_main(click_ctx, verbose, log_file, no_prompt, token_path):
 @usage_flag(help_message="Show the usage for available projects, in GBHours and cost.")
 @users_flag(help_message="Display users associated with a project(Requires a project id).")
 @click.option("--projects", "-lp", is_flag=True, help="List all project connected to your account.")
+@click.option("--all", is_flag=True, help="List all project connected to your account.")
```

Comment:
Maybe this instead to not override the builtin `all`? 
```suggestion
@click.option("--include-inactive", is_flag=True, help="List all project connected to your account.")
```
or 
```suggestion
@click.option("--show-all", is_flag=True, help="List all project connected to your account.")
```